src/Parser/Hunter/Marksmanship/Modules/Spells/Trueshot.js

```diff
@@ -109,6 +113,14 @@ class Trueshot extends Analyzer {
           .regular(recommended - 10)
           .major(recommended - 20);
       });
+    when(this.executeTrueshots).isLessThan(1)
+      .addSuggestion((suggest, actual, recommended) => {
+      return suggest(<span>You should make sure to have atleast 1 <SpellLink id={SPELLS.TRUESHOT.id}/> cast during execute (where you are buffed by <SpellLink id={SPELLS.BULLSEYE_TRAIT.id}/>) to get the most out of <SpellLink id={SPELLS.TRUESHOT.id}/> as possible.</span>)
```

Comment:
"to get the most out of Trueshot as possible" doesn't make sense. Should be either "to get the most out of Trueshot", or "to get as much out of Trueshot as possible".

Reply:
Agreed